### PR TITLE
Rasta hat

### PR DESCRIPTION
**Pull request by ChilbroBaggins**
I had some goofy stuff I was messing around with. I wanted to clear it all out though before updating my master. So, if you think it's fun merge it. 
Screenshot 2022-08-15 101528
Screenshot 2022-08-15 101521
Screenshot 2022-08-15 101516

### CODE REVIEW

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
Probably shouldn't add content that has no way of being acquired in game.

**Review by mirrorcult — CHANGES_REQUESTED** [review]
see above

File: Resources/Prototypes/Entities/Clothing/Head/hats.yml [hunk 1]
```diff
@@ -467,3 +467,14 @@
     sprite: Clothing/Head/Hats/nursehat.rsi
   - type: Clothing
     sprite: Clothing/Head/Hats/nursehat.rsi
+
+- type: entity
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Head/hats.yml:471]: a

**Comment by ChilbroBaggins** [comment]
OK, added to the Clothesmate and Autodrobe.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**